### ISSUE TITLE
Review services_config/test WMCore services configuration

### ISSUE THREAD

**Issue by amaltaro** [dmwm/WMCore#11084]
**Impact of the new feature**
All WMCore services

**Is your feature request related to a problem? Please describe.**
As we move our dev/test activities to the kubernetes cmsweb-testX clusters, we need to review the services configuration and make sure that everything is properly configured and isolated from production and pre-production activities/database.

**Describe the solution you'd like**
Review all the services configuration and ensure that they won't talk to (pre-)production services unless it's deemed necessary.
Services that are performing real actions might be switched off (e.g., microservices).

On what concerns CouchDB, we can either (a new issue will be open for this):
a) use a single node (VM?) running CouchDB with all the databases, and sharing it among all the cmsweb-testX clusters; or
b) resume attempts to deploy CouchDB with kubernetes, even if that is still our CouchDB 1.6.1 RPM-based package.

Deliverable for this issue is a new services_config pull request against the `test` cluster. Note that secrets might have to be discussed privately with the HTTP team.

**Describe alternatives you've considered**
This must be done before we actually start using those cmsweb-testX clusters for WMCore validations.

**Additional context**
Target gitlab repository is: https://gitlab.cern.ch/cmsweb-k8s/services_config

**Comment by amaltaro** [comment]
This has been fixed by: https://gitlab.cern.ch/cmsweb-k8s/services_config/-/merge_requests/154

Thanks for following this up, Erik! If there is anything pending here, please reopen this issue.
